README.md

```diff
@@ -320,17 +297,25 @@ the market's order book in sorted lists or as a depth chart. You can place your
 order and monitor it's status in the same market view. The GUI application is
 managed by the **dexc** utility in *client/cmd/dexc*.
 
+If you modify any of the HTML templates in *client/webserver/site/src/html*, it
+is necessary to regenerate the localize templates in
+*client/webserver/site/src/localized_html*:
+
+```
+go generate ./...
```

Comment:
I just moved this down here to make the "Build from Source" section short and sweet, but I think this whole "Client Applications and the Core Package" section can also leave the README and live in the wiki instead.

Reply:
I think all the developer stuff is better in the wiki personally.  I can find a spot in the wiki to dump this section so it doesn't get lost.